iroha/src/queue.rs

```diff
@@ -13,6 +13,11 @@ pub struct Queue {
 }
 
 impl Queue {
+    /// Get cloned transactions that are currently in a queue.
+    pub fn pending_transactions(&self) -> Vec<AcceptedTransaction> {
+        self.pending_tx_by_hash.values().cloned().collect()
+    }
```

Comment:
Looks like we should transform Accepted Transactions into Pending Transactions here, not in Torii State